components/Code/index.jsx

```diff
@@ -4,8 +4,8 @@ const PropTypes = require('prop-types');
 // Only load CodeMirror in the browser, for SSR
 // apps. Necessary because of people like this:
 // https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
-const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
-const canonicalLanguage = require('@readme/syntax-highlighter/canonical');
+const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter').default : false;
```

Comment:
Now that the server-compatible version of this,  `index.node.js` is the `main` entrypoint, you might need to reference the client version.

Here's `main`: https://github.com/readmeio/syntax-highlighter/blob/master/src/index.node.js (no default).
Here's the client: https://github.com/readmeio/syntax-highlighter/blob/master/src/index.js (default)

Reply:
Not sure what I need to change here.

Reply:
@kellyjosephprice You mind taking this? I'm not sure what exactly this require needs to be changed to.

Reply:
Sure!

Reply:
I believe there's no change needed? It looks like syntax-highlighter is using the `browser` field in `package.json` and the webpack build for markdown is loading the correct file.

```sh
rg SyntaxHighlighter dist
dist/main.node.js
5057:  }), console.log("SyntaxHighlighter loading server");

dist/main.js
20852:  console.log("SyntaxHighlighter loading client");
```